### PR TITLE
Quieter internal logging, and add presenter for config client

### PR DESCRIPTION
**Pull request by mikowitz**
* Move a bunch of internal client logging on the happy client init path to `DEBUG` rather than `INFO`
* Add `ConfigClientPresenter` to format an `INFO` log on successful ConfigClient init

### CODE REVIEW

File: lib/prefab/config_client_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    def initialize(size:, source:, project_id:, project_env_id:, api_key:)
+      @size = size
+      @source = source
+      @project_id = project_id
+      @project_env_id = project_env_id
+      @api_key = api_key
+    end
+
+    def to_s
+      "Configuration Loaded count=#{@size} source=#{@source} project=#{@project_id} project-env=#{@project_env_id} prefab.api-key-id=#{@api_key}"
```
Comment (jdwyah) [inline lib/prefab/config_client_presenter.rb:14]: Is the api_key here just the ID? or the whole thing. We shouldn't print the api key.

**Review by jdwyah — CHANGES_REQUESTED** [review]
besides comment, this lg

File: lib/prefab/config_client_presenter.rb (hunk 1 above)
Comment (mikowitz) [reply lib/prefab/config_client_presenter.rb]: ok, updated to extract the id from the API key.

**Review by jdwyah — APPROVED** [review]
👍

**Pull request merged**